pkg/config/db.go

```diff
@@ -0,0 +1,156 @@
+package config
+
+import (
+	v1API "github.com/supabase/cli/pkg/api"
+	"github.com/supabase/cli/pkg/cast"
+	"github.com/supabase/cli/pkg/diff"
+)
+
+type (
+	PoolMode string
```

Comment:
Let's move the entire enum declaration here https://github.com/supabase/cli/blob/0377d4730f5f9ba3a8c7c64cfef37584a9d82150/pkg/config/config.go#L60-L65